src/natcap/invest/__init__.py

```diff
@@ -34,6 +32,11 @@
 def set_locale(locale_code):
     """Set the `gettext` attribute of natcap.invest.
 
+    This is the locale that will be used for translation. The `gettext`
```

Comment:
Possibly something amiss with the merge? This looks correct but I'm surprised this file is in the diff